shell/browser/api/electron_api_app.cc

```diff
@@ -1276,10 +1276,16 @@ std::vector<gin_helper::Dictionary> App::GetAppMetrics(v8::Isolate* isolate) {
     auto pid_dict = gin_helper::Dictionary::CreateEmpty(isolate);
     auto cpu_dict = gin_helper::Dictionary::CreateEmpty(isolate);
 
-    double usage =
-        process_metric.second->metrics->GetPlatformIndependentCPUUsage()
-            .value_or(0);
-    cpu_dict.Set("percentCPUUsage", usage / processor_count);
+    // Default usage percentage to 0 for compatibility
+    double usagePercent = 0;
+    if (auto usage = process_metric.second->metrics->GetCumulativeCPUUsage()) {
```

Comment:
This needs to use the base::TimeDelta value. A gin converter ([see time_converter.cc](https://github.com/electron/electron/blob/main/shell/common/gin_converters/time_converter.cc)) might need to be added for the JS binding, if it's not set as a primitive type.

Reply:
@rcombs are you still interested in moving this forward?

Reply:
Looks like this is no longer needed after the build failures were resolved ✅